README.md

```diff
@@ -1,3 +1,142 @@
+This repository contains an up-to-date fork of
+[Wasmtime](https://docs.wasmtime.dev/), a standalone WebAssembly engine. This
+fork adds support for the [WasmFX](https://wasmfx.dev/) proposal for stack
+switching. 
+
+The [stack switching repository](https://github.com/WebAssembly/stack-switching) contains a 
+[high-level summary of the proposal](https://github.com/WebAssembly/stack-switching/blob/main/proposals/continuations/Explainer.md),
+[a more formal description](https://github.com/WebAssembly/stack-switching/blob/main/proposals/continuations/Overview.md),
+and [examples](https://github.com/WebAssembly/stack-switching/tree/main/proposals/continuations/examples).
+
+## Building 
+
+The build steps are equivalent to the [standard steps for building Wasmtime from
+source](https://docs.wasmtime.dev/contributing-building.html), but using this
+repository instead. There is no need to build or install the original version of
+Wasmtime to use this fork.
+
+Concretely, the steps are as follows:
+
+1. Make sure that you have a Rust toolchain installed, for example using
+   [rustup](https://www.rust-lang.org/tools/install).
+2. Check out this repository:
+``` sh
+git clone https://github.com/wasmfx/wasmfxtime.git
+cd wasmfxtime
+git submodule update --init
+```
+3. Build:
+``` sh
+cargo build
+```
+
+As a result, a debug build of the `wasmtime` executable will be created at
+`target/debug/wasmtime`.
+
+To create a release build instead, run `cargo build --release`, which will
+create `target/release/wasmtime`.
+
+
+## Running programs
+
+A WebAssembly module `my_module.wat` (or `my_module.wasm`) is executed using the
+`wasmtime` executable [in the usual way](https://docs.wasmtime.dev/cli.html). To
+run programs containing WasmFX instructions, enable the necessary features as
+follows:
+
+``` sh
+wasmtime -W=exceptions,function-references,typed-continuations my_module.wat
```

Comment:
is the `run` subcommand the default command?

Reply:
Yes